utils/keyboard-shortcut.js

```diff
@@ -0,0 +1,29 @@
+export const ALT = 'alt';
+export const CONTROL = 'mod';
```

Comment:
Dunno how we want to name these, and/or if we want aliases so `CONTROL` and `META` do the same thing. 🤷‍♂️